### ISSUE TITLE
about my own database

### ISSUE THREAD

**Issue by an1018** [JiahuiYu/generative_inpainting#187]
@JiahuiYu  I have a question to consult you.My training set has 5000 pictures. It performs very well in training set and very poorly in test set. How to deal with this situation?
